gapic-generator/lib/gapic/generators/default_generator.rb

```diff
@@ -77,6 +79,10 @@ def generate gem_presenter: nil
         files << g("gem/yardopts.erb",    ".yardopts",                    gem: gem)
         files << g("gem/license.erb",     "LICENSE.md",                   gem: gem)
         files << g("gem/entrypoint.erb",  "lib/#{gem.name}.rb",           gem: gem)
+        files << generate_nontemplate_file(
+          filename: "gapic_metadata.json",
+          content: JSON::pretty_generate(gem.first_package_drift_manifest)
```

Comment:
Maybe consider just creating a template with this logic, so we don't need a separate generation method.

Reply:
In fact, what we'd do elsewhere is actually create another presenter method, say, `first_package_drift_manifest_json`, which does the JSON pretty_generate, and then just render that directly in a template.

Reply:
What benefits does the template layer bring to justify routing non-template files through it?

Reply:
fixed